src/app/components/modals/modal-highlight/modal-highlight.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="modal-highlight-container">
+  <button type="button" class="close pull-right" aria-label="Close" (click)="bsModalRef.hide()">
```

Comment:
There's something weird with this button, take a look at my GIF please: https://gyazo.com/bc2c8c9b62a54553406c87c7d516896e
Can you look into fixing this?

I tried clicking on the "X" button to discard the modal, at first I thought it did not work but after a few tries I found out that it actually does work but the size of the button is just off